misc-scripts/xref_mapping/XrefParser/MIMParser.pm

```diff
@@ -21,162 +21,274 @@ package XrefParser::MIMParser;
 
 use strict;
 use warnings;
+
 use Carp;
-use POSIX qw(strftime);
 use File::Basename;
```

Comment:
File::Basename can probably be safely removed

Reply:
Done...